client/lib/plans/features-list.js

```diff
@@ -1314,6 +1328,67 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'Live chat support 24X7' ),
 		getDescription: () => i18n.translate( 'Live chat is available 24/7.' ),
 	},
+
+	/* START - Jetpack tiered product-specific features */
+	[ FEATURE_JETPACK_REAL_TIME_BACKUPS ]: {
+		getSlug: () => FEATURE_JETPACK_REAL_TIME_BACKUPS,
+		getTitle: () =>
+			i18n.translate( 'Real-time backups {{noHighlight}}(as you edit){{/noHighlight}}', {
+				components: {
+					/* eslint-disable-next-line wpcalypso/jsx-classname-namespace */
+					noHighlight: <span className="not-highlighted" />,
+				},
+			} ),
+		isHighlighted: () => true,
+	},
+	[ FEATURE_JETPACK_CLOUD_BASED_BACKUPS ]: {
+		getSlug: () => FEATURE_JETPACK_CLOUD_BASED_BACKUPS,
+		getTitle: () => i18n.translate( 'Backups are cloud based' ),
+	},
+	[ FEATURE_JETPACK_20GB_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_JETPACK_20GB_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( '20GB of backup storage' ),
+	},
+	[ FEATURE_JETPACK_1TB_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_JETPACK_1TB_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( '1TB of backup storage' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( '1TB of storage' )` *ES Score: 7*